os/drivers/pm/pm.c

```diff
@@ -130,6 +131,10 @@ static int pm_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			}
 		}
 		break;
+	case PMIOC_START:
+		pm_start();
```

Comment:
For existing compatibility, case PMIOC_RESUME: also needs a temporary modification to call pm_start when it is 0.

Reply:
Done